src/components/select/select.component.html

```diff
@@ -66,13 +66,12 @@
     <div class="ux-select-icons">
         <i *ngIf="clearButton && allowNull && value"
            uxFocusIndicator
-           tabindex="0"
+           [attr.tabindex]="disabled ? null : 0"

```

Comment:
Probably best to use `-1` as the value when making an item not-tabbable as a value of `null` will use the default tabindex for that element, so for example an `a` tag or `button` element would be tabbable with a value of `null`. So for the time being this would make it not tabbable, should we decide to change this to use a button element or something this issue would then reappear, so its just safer to use `-1` instead.

Reply:
Updated to -1